data/resources/style.css

```diff
@@ -159,30 +159,61 @@ messagebubble.document {
 }
 
 messagebubble.document .file {
-  margin: 6px 0;
+  margin: 3px 0;
+}
+
+messagebubble.document .sender-text-red,
+.sender-text-orange,
+.sender-text-violet,
+.sender-text-green,
+.sender-text-cyan,
+.sender-text-blue,
+.sender-text-pink {
+  margin-bottom: 3px;
+}
+
+messagebubble.document .file > overlay {
+  transition: 200ms cubic-bezier(0.25, 0.46, 0.45, 0.94);
```

Comment:
It's used when thumbnail loaded and small circle transforms into big square

Reply:
Video for example

[Screencast from 2022-11-20 18-29-32.webm](https://user-images.githubusercontent.com/63008755/202910852-4cc66e9c-59c7-4f0e-bac5-805ded9de806.webm)

Reply:
What about showing the minithumbnail while the higher resolution thumbnail is downloading?

Reply:
I'm doing the same in #192 for photos and gifs.

Reply:
Ok, TIL that gtk supports blur. maybe it will look good

Reply:
> Ok, TIL that gtk supports blur. maybe it will look good

Yep, the idea is to use it at some point for minithumbnails, but for now I'd like to keep things simpler and not using it.

Reply:
Without blur minithumbnail looks very strange
![image](https://user-images.githubusercontent.com/63008755/202915739-c0274e84-4465-42f6-97cb-274457dccb65.png)

Reply:
> Without blur minithumbnail looks very strange

I know, but that's temporary. I'd like to create a more general widget in the future for showing thumbnails + adding blur + transitions to the higher resolution one.

Also, the minithumbnail will be only shown when downloading the full thumbnail, so you probably won't even notice it.